lib/mapper.js

```diff
@@ -108,6 +113,53 @@ function formatCompany(company){
   return ret;
 }
 
+ /**
+ * Formats location data for use with intercom
+ *
+ * https://doc.intercom.io/api/#user-model
+ *
+ * @param {Object} location
+ * @return {Object}
+ * @api private
+ */
+
+function formatLocation(context){
```

Comment:
Break this and any supporting functions into a separate file and write unit tests for it